src/Test/nlp.jl

```diff
@@ -174,12 +188,22 @@ end
 
 function MOI.features_available(d::FeasibilitySenseEvaluator)
     if d.enable_hessian
-        return [:Grad, :Jac, :Hess]
+        return [:Grad, :Jac, :Hess, :ExprGraph]
     else
-        return [:Grad, :Jac]
+        return [:Grad, :Jac, :ExprGraph]
     end
 end
 
+function MOI.objective_expr(d::FeasibilitySenseEvaluator)
+    return :(0 * x[1])
```

Comment:
This seems hacky. What about returning 0.0 instead?

Reply:
I think there is a bug somewhere if you pass 0.0. It's on my list of things to investigate today.

Reply:
worth merging in master and trying again?